src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,53 @@
+.catalog {
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+}
+
+@media (max-width:488px) {
+  .catalog {
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(1, 200px);
+    gap: 48px;
+  }
+}
+
+@media (min-width:489px) and (max-width:768px) {
+  .catalog {
```

Comment:
It is not recommended to include a selector within a media query in the SCSS.

.catelog {
  width: 100px;
  
  @media (...) {
    width: 200px;
  }
}